meetings/archive/20241204_agenda_and_minutes.md

```diff
@@ -30,8 +31,22 @@ X people in total
 
 ## Announcements
 
-- [ ]
+- [x] JRG: Open votes for two CEPs, til 2024-12-18:
+    - [Support for ABI3 packages](https://github.com/conda/ceps/pull/86)
+    - [Standardize algorithm for directory hashing](https://github.com/conda/ceps/pull/100)
+- [x] Reminder: Today's is the last meeting of 2024. We'll cancel 2024-12-18 + 2025-01-01. See you Jan 15th!
 
 ## New Agenda Items
 
-- [ ]
+- [x] JRG: Low engagement in https://conda.zulipchat.com. Join us! :)
+- [x] JRG/JL: Github App / bot for votes: https://github.com/conda/governance/issues/195.
+- [x] WV: shell completion scripts
+    - Would like to implement something similar to Homebrew's standardized location for shell completion scripts
+    - https://github.com/prefix-dev/pixi/issues/2366
+    - CHL: as a meta-question, should we write something similar to the [Linux Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html) for the conda ecosystem?
+        - Generally, sure. But Windows...
+        - But writing this/these standard(s) should not block any on-going/new work. This exercise is mostly a backfill.
+- [x] WV: rattler-build cache
+- [x] WV: pixi build
+- [x] DH: conda-pupa
```

Comment:
I could not attend this meeting and I understood almost all the topics but this one from the notes. What is pupa?

Reply:
This is @dholth's project https://github.com/dholth/conda-pupa

Reply:
Reminds me of `conda pipbuild`, which did a similar thing in the past before being deprecated/removed

It would be good to incorporate the learnings from that experience into this tool

Reply:
@jaimergp @dholth @jjhelmus @lwinfree ^ FYI

Reply:
There's also https://github.com/ContinuumIO/pypi-conda-builds 👀

Reply:
We need to mark this conversation as resolved to merge :) Let me know if you got the info you needed @ocefpaf. Note I also added links to other items in the agenda.

Reply:
Thanks Jaime. Sorry for holding the merge. I'm fine with the extra info added. Thanks again!

Reply:
Interesting. Didn't realize we required to resolve conversations to merge. Maybe it is worth relaxing this constraint given it can be easier to find past discussions if the conversations remain open